ts/components/conversation/message/message-content/MessageContent.tsx

```diff
@@ -181,26 +213,29 @@ export const MessageContent = (props: Props) => {
 
   const bgShouldBeTransparent = isShowingImage && !hasText && !hasQuote;
   const toolTipTitle = moment(serverTimestamp || timestamp).format('llll');
+  // tslint:disable: use-simple-attributes
 
   return (
-    <div
+    <StyledMessageContent
       className={classNames(
         'module-message__container',
         `module-message__container--${direction}`,
         bgShouldBeTransparent
           ? `module-message__container--${direction}--transparent`
           : `module-message__container--${direction}--opaque`,
-        firstMessageOfSeries || props.isDetailView
-          ? `module-message__container--${direction}--first-of-series`
-          : '',
-        lastMessageOfSeries || props.isDetailView
-          ? `module-message__container--${direction}--last-of-series`
-          : '',
+
         flashGreen && 'flash-green-once'
       )}
-      style={{
-        width: isShowingImage ? width : undefined,
-      }}
+      style={
+        {
+          width: isShowingImage ? width : undefined,
+          '--radius-lg': '18px',
```

Comment:
Is this here because of the perforance stuff you were talking about? I feel like if we are using these border radius values everywhere they should be in SessionTheme.tsx so they are easily accessible for future components?

Reply:
not really for performance here. I just didn't want to copy paste those values 10 times up there
```ts
  border-bottom-left-radius: ${props =>
    !props.isOutgoing
      ? props.lastOfSeries
        ? 'var(--radius-lg)'
        : 'var(--radius-sm)'
      : 'var(--radius-lg)'};
`;
```

I think it makes sense to move stuff in the SessionTheme.tsx file when they are needed. For now, those values are only used here so I'd rather keep them here. It would just clutter more the SessionTheme file for a value used very locally.

What do you think?

Reply:
Actually I was thinking we could make them variables within the file and then just use them directly in the styled component instead of setting them in the Style Object (which is loaded every render). Like this.

```tsx
const radiusLg = 'XXpx';
const radusSs = 'XXpsx'

const StyledMessageContent = styled.div<{
  isOutgoing: boolean;
  firstOfSeries: boolean;
  lastOfSeries: boolean;
}>`
  border-top-right-radius: ${props =>
    props.isOutgoing
      ? props.firstOfSeries
        ? `${radiusLg}`
        : `${radiusSm}`
      : `${radiusLg}`};
// ... more code here
```

Reply:
updated!